pkg/app.go

```diff
@@ -42,6 +44,7 @@ func (a *App) setRouters() {
 
 	a.Router.HandleFunc("/courses", a.passDB(handlers.GetCourses)).Methods("GET")
 	a.Router.HandleFunc("/course/{code}", a.passDB(handlers.GetCourse)).Methods("GET")
+	a.Router.Handle("/group/get", auth.Authenticator(a.passDB(handlers.GetGroups), a.HMACSecret)).Methods("GET")
 }
 
 func (a *App) passDB(handler func(db *sql.DB, w http.ResponseWriter, r *http.Request)) http.HandlerFunc {
```

Comment:
need not be in this PR, but passDB should also use context now

Reply:
Will create a new PR for this, I don't want to go beyond the scope of the PR.